social/backends/vend.py

```diff
@@ -0,0 +1,84 @@
+"""
+Vend  OAuth2 backend:
+
+"""
+from requests import HTTPError
+from social.backends.oauth import BaseOAuth2
+from social.exceptions import AuthCanceled, AuthUnknownError
+
+
+class VendOAuth2(BaseOAuth2):
+    name = 'vend'
+    AUTHORIZATION_URL = 'https://secure.vendhq.com/connect'
+    ACCESS_TOKEN_URL = ''
+    SCOPE_SEPARATOR = ' '
+    REDIRECT_STATE = False
+    REDIRECT_URI_PARAMETER_NAME = 'redirect_uri'
+    EXTRA_DATA = [
+                     ('refresh_token', 'refresh_token'),
+                     ('domain_prefix','domain_prefix')
+    ]
+    def get_user_id(self, details, response):
+        return None
+    def get_user_details(self, response):
+        return {}
+
+    def user_data(self, access_token, *args, **kwargs):
+
+        return None
+
+
+    def access_token_url(self):
+        return self.ACCESS_TOKEN_URL
```

Comment:
This is the default behavior, why override? I see that it requires a ``domain_prefix`` a few lines below, if that's the case, then that logic should be put here.